dbt_project.yml

```diff
@@ -61,13 +64,15 @@ models:
   snowplow_web:
     +materialized: view
     base:
+      manifest:
+        +schema: "snowplow_manifest"
       scratch:
         +schema: "scratch"
         +tags: "scratch"
         bigquery:
           enabled: "{{ target.type == 'bigquery' | as_bool() }}"
-        redshift:
-          enabled: "{{ target.type == 'redshift' | as_bool() }}"
+        default:
+          enabled: "{{ target.type in ['redshift', 'postgres'] | as_bool() }}"
```

Comment:
Is this good practise? I appreciate Redshift and Postgres are the same right now... but they could deviate in the future, either at the warehouse level or the snowplow structure level. I fear we're setting ourselves up for needing a future fix by treating them as the same.

Reply:
Fair point. My thinking here:
* If Postgres changes at a warehouse level then dbt should be handling a lot of those changes under the hood e.g. changes in DDL syntax etc.
* If the Snowplow structure changes then we can split out the Redshift and Postges models where appropriate. Throughout the package I have tried to consolidate the models across the various warehouses where possible in order to remain DRY. This seems to be pretty standard practice in dbt packages.
* If and when we do have to have distinct models for Postgres and Redshift, assuming the resulting table schema is the same, the change will not impact the package user. The model name will still be the same.

Reply:
Good points and certainly good enough for me.